src/__tests__/host-component-names.test.tsx

```diff
@@ -87,7 +87,7 @@ describe('configureHostComponentNamesIfNeeded', () => {
       .toThrowErrorMatchingInlineSnapshot(`
       "Trying to detect host component names triggered the following error:
 
-      Unable to find an element with testID: text
+      \`render\` method has not been called
```

Comment:
This error message changing is an unintended side effect. Basically when the `getByTestID` call fails in this test, it tries to print the DOM now, and `screen.toJSON()` fails because we haven't actually called render. 

I'm not sure if we would get this error in the real world or if it even matters that the message changed, since it didn't seem like a message that would have made sense to the user in the first place (the `testID=text` code is all internal to this library).